### PR TITLE
[sentry-breakpad/0.5.4] Bump version

### PR DESCRIPTION
**Pull request by MartinDelille**
---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/sentry-breakpad/all/conanfile.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def requirements(self):
 
     def validate(self):
         if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, 11)
+            tools.check_min_cppstd(self, tools.Version(self.version) < "0.5.4" ? 11 : 17)
```
Comment (RubenRBS) [inline recipes/sentry-breakpad/all/conanfile.py:41]: Oops :) Python's way is:

```suggestion
            tools.check_min_cppstd(self, 11 if tools.Version(self.version) < "0.5.4" else 17)
```

**Review by RubenRBS — CHANGES_REQUESTED** [review]
Quick fix for this

File: recipes/sentry-breakpad/all/conanfile.py [hunk 2]
```diff
@@ -26,6 +27,18 @@ class SentryBreakpadConan(ConanFile):
         "fPIC": True,
     }
 
+    @property
+    def _min_cppstd(self):
+        return 11 if Version(self.version < "0.5.4") else 17
```
Comment (SpaceIm) [inline recipes/sentry-breakpad/all/conanfile.py:32]:
```suggestion
        return 11 if Version(self.version) < "0.5.4" else 17
```

File: recipes/sentry-breakpad/all/conanfile.py [hunk 3]
```diff
@@ -26,6 +27,18 @@ class SentryBreakpadConan(ConanFile):
...
+
+    @property
+    def _compilers_minimum_version(self):
+        return {} if Version(self.version < "0.5.4") else {
```
Comment (SpaceIm) [inline recipes/sentry-breakpad/all/conanfile.py:36]:
```suggestion
        return {} if Version(self.version) < "0.5.4" else {
```

File: recipes/sentry-breakpad/all/conanfile.py [hunk 4]
```diff
@@ -26,6 +27,18 @@ class SentryBreakpadConan(ConanFile):
...
+            "gcc": "7",
+            "clang": "7",
+            "apple-clang": "10",
```
Comment (SpaceIm) [inline recipes/sentry-breakpad/all/conanfile.py:39]:
```suggestion
            "apple-clang": "10",
            "Visual Studio": "15",
            "msvc": "191",
```

File: recipes/sentry-breakpad/all/conanfile.py [hunk 5]
```diff
@@ -1,6 +1,7 @@
 from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
 from conan.tools.apple import is_apple_os
+from conan.tools.microsoft import check_min_vs, is_msvc
```
Comment (SpaceIm) [inline recipes/sentry-breakpad/all/conanfile.py:4]:
```suggestion
```

**Review by SpaceIm — APPROVED** [review]

**Review by jwillikers — APPROVED** [review]

**Comment by MartinDelille** [comment]
@RubenRBS Is it fine for you ?

**Review by SSE4 — APPROVED** [review]

**Comment by RubenRBS** [comment]
Sorry about the delay, lgtm :)

**Pull request merged**